### DISCUSSION TITLE
Cyber Security study guide

### DISCUSSION THREAD

**Discussion by ghost** [community/community#47079] [Other Feature Feedback, Questions, & Ideas]
<p align="justify">If you are interested in pursuing a cybersecurity education track, here is a general roadmap you can follow:</p>

<ol align="justify">
<li>Information Technology Fundamentals: Start by learning about the basics of information technology, including hardware, software, networking, and operating systems.</li>

<li>Information Security: Learn about major threats to information security, including viruses, spyware, phishing, and other types of cyberattacks.</li>

<li>Cryptography: Study cryptography and its concepts, including cipher, public and private key, digital signature and other methods of information security.</li>

<li>Intrusion Detection Systems (IDS): Learn about intrusion detection systems (IDS) and how they are used to protect networks and systems from attacks.</li>

<li>Firewalls and Network Security: Learn about firewalls and their concepts, including security rules, packet filtering, and other network security techniques.</li>

<li>Web Application Security: Study about web application security, including common vulnerabilities such as SQL injection and XSS attacks, and how to protect applications from these threats.</li>

<li>Cloud Security: Learn about cloud security, including security configuration, monitoring, and risk management in cloud environments.</li>

<li>Incident Response and Forensics: Learn about incident response and forensics, including how to identify, investigate, and respond to information security incidents.</li>

<li>Compliance and Regulations: Study the regulations and compliance standards related to information security, including PCI DSS, ISO 27001 and other relevant regulations.</li>

<li>Practice and Certifications: Practice what you've learned on real projects and consider obtaining information security certifications such as Certified Information Systems Security Professional (CISSP) or Certified Ethical Hacker (CEH).</li>
</ol>

<p>Remember that cybersecurity is an ever-evolving field, so it's important to keep up with it.</p>

**Comment by Akash1134** [comment]
Hi @alexandresantosal91 

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by Kurtchen1a** [comment]
Our education partner the Swiss Cyber Institute is a good starting point for Cyber Security education. In this context some insights about a Cyber Career could be from interest https://cyberunity.io/en/why-cyber-security-is-a-good-career-in-switzerland/
